detection-rules/body_extortion.yml

```diff
@@ -8,22 +8,16 @@ severity: "low"
 source: |
   type.inbound
   and (
-  any(ml.nlu_classifier(body.current_thread.text).intents,
-      .name == "extortion" and .confidence == "high"
-  )
-  and any(ml.nlu_classifier(body.current_thread.text).entities, .name == "financial")
-  )
-  and (
-    (
-      sender.email.domain.root_domain in $free_email_providers
-      and sender.email.email not in $sender_emails
+    any(ml.nlu_classifier(body.current_thread.text).intents,
+        .name == "extortion" and .confidence == "high"
     )
-    or (
-      sender.email.domain.root_domain not in $free_email_providers
-      and sender.email.domain.domain not in $sender_domains
+    and any(ml.nlu_classifier(body.current_thread.text).entities,
+            .name == "financial"
     )
-    // many extortion emails spoof sender domains and fail sender authentication
-    or any(headers.hops, .authentication_results.dmarc == "fail")
+  )
+  and (
+      not profile.by_sender().any_false_positives
+      and not profile.by_sender().solicited
```

Comment:
i think this opens us up to FPs a little bit, do we wanna add prevalence?

Reply:
Yeah Im not done with this one yet. I started some hunts to flesh it out